### DISCUSSION TITLE
Moving Data from one bar to another via export error

### DISCUSSION THREAD

**Discussion by joshuathejack** [karlomikus/bar-assistant#381] [Q&A] [answered]
I'm doing some clean up on my old docker instances and have created an "export" in my old Bar Assistant Bar.  I'd like to import this zip to my new Bar Assistant docker but there isn't any import feature in the UI and the instructions to import via CLI always give me a "no configuration file provided: not found".  Am I missing something?

**Comment by zhdenny** [comment, marked as answer]
I give noob friendly instructions on how to import my bar which I share with anyone who wants it. I bet you can the information there to figure it out for your own bar.

https://github.com/zhdenny/bar_assistant_database

Also, be sure you are using the exact same versions to Export/Import. A Bar exported from one App Version is not compatible for import into a different App version.

**Reply by joshuathejack** [reply]
Thank you!  Definitely a noob to containerization, so I have a very noob follow up.  You said to drop the ZIP in my Appdata directory.  How do I find out where that is?  Is it in a mapped volume?  Or is it located in the host system?  Again apologies.  I believe I understand shelling into the container (I use Portainer and this makes it pretty easy).  And the rest of your steps are very well documented.  Thanks ahead of time.

**Comment by zhdenny** [comment]
Step 2
```
Place database zip in your bar-assistant appdata folder. Be sure permissions for database zip file match your bar-assistant appdata (should be 33:33 as of V4 of BarAssistant)
```

Take a look at my docker_compose file in my Github Repo [here](https://github.com/zhdenny/bar_assistant_database/blob/75e8d533678cbc8e43434bd5ec5ec2a918054113/docker_compose#L41).
For the "bar_assistant" container, I have mapped a local location `/mnt/user/appdata/bar-assistant` to a docker container location `/var/www/cocktails/storage/bar-assistant`. The local location is the bar assistant appdata location. The instructions are telling you to put the zip file in the local location. When I say "local" I am referring to a location that is directly accessible on your server. When I refer to a docker location, that is a location as seen through the docker container filesystem

**Comment by karlomikus** [comment]
If you are using named volumes the easiest way to get the zip file into container is via [docker copy command](https://docs.docker.com/reference/cli/docker/container/cp/).

And if you are using bind mounts you can put the file into locally mapped location as described by zhdenny above.
